ngraph/core/include/ngraph/opsets/opset.hpp

```diff
@@ -133,4 +133,14 @@ namespace ngraph
     const NGRAPH_API OpSet& get_opset3();
     const NGRAPH_API OpSet& get_opset4();
     const NGRAPH_API OpSet& get_opset5();
+
+    inline const NGRAPH_API std::array<std::reference_wrapper<const ngraph::OpSet>, 5>
```

Comment:
@ilyachur is it ok to extend opset API ? I think it can be generally useful so I putted it here.

Reply:
I am not sure that this method is needed for external user.... Why do we need this method in public API? Looks like at the current moment we have only one place when we use this order.

Reply:
Ok, will move to my private scope.

Reply:
@ilyachur I was too quick with my answer :)  now I recalled why i wanted this function in opset module - because if we add new opset in the future no one will remember to update also serialization module; so I figure it would be the best if opset module contain function which will return all of the opsets (which I need in serialization); maybe it's not needed for external user but code will be more coherent if it's there

Reply:
But in any case, you also can forget about this function if you add a new opset. It doesn't matter where we have this method.
But if it is required only for our internal usage we shouldn't add it to public API. Moreover we control our code, and if we add a new opset we need to update such method (we already have the same issue [here](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/readers/ir_reader/ie_ir_parser.cpp#L67))

Reply:
Ok, moved to private in this [commit](https://github.com/openvinotoolkit/openvino/pull/2579/commits/87bd9a8c76c41d15d308ba501a7bf144cfe961a4).

Reply:
Github was not showing this comments appropriately and I missed it at Friday. @ilya-lavrenov could you reprhase your question? I'm not sure what's your point here.  

FYI @ilyachur 

![image](https://user-images.githubusercontent.com/56125763/97150220-191b6800-176e-11eb-89be-7e4cb1b3a485.png)

Reply:
Ok, I talked with ilyachur and I'm assuming there won't be any changes here. If we need further discussion please reopen.